database/seeds/BlogTableTestAddBlogSeeder.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+define("_BLOG_TEST_RECORD_NUMBER", 5);
+define('_UserCanSeeId', 1);
+define('_UserCanDeleteId', 2);
+define('_UserCanNotSeeId', 3);
+define('_UserCanNotDeleteId', 4);
+
+class BlogTableTestAddBlogSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        //add data for user can see
+        DB::table('blogs')->insert([
+            'user_id' => _UserCanSeeId,
+            'title' => 'user can see title1',
```

Comment:
Don't use magic number in code.
you should rename title1,2,3. 
Ex: user can see title of row has Id 1.

Reply:
I fixed